content-src/asrouter/templates/ReturnToAMO/_ReturnToAMO.scss

```diff
@@ -0,0 +1,131 @@
+.ReturnToAMOOverlay,
+.amo + body.hide-main {  // sass-lint:disable-line no-qualifying-elements
+  background: $white;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  display: block;
+  z-index: 2100;
+
+  .ReturnToAMOOuterContainer {
+    width: 835px;
+    height: 400px;
+    position: fixed;
```

Comment:
Any reason you didn't want to use `flex`/ center this? Is the spec supposed to be fixed to the top?